src/main/java/io/galeb/manager/entity/Account.java

```diff
@@ -121,4 +121,10 @@ public Account setPassword(String password) {
         this.password = ENCODER.encode(password);
         return this;
     }
+
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?